frontend/src/components/statusCards/StatusCardDroppedOut.vue

```diff
@@ -43,7 +43,7 @@ export default {
     }
   },
   methods: {
-    ...mapActions([ 'updateStudentStatus' ]),
+    ...mapActions([ 'updateStudentStatus']),
```

Comment:
(just nitpicking here)

```suggestion
    ...mapActions(['updateStudentStatus']),
```